File: src/2806.account-balance-after-rounded-purchase.2955/solution.ts
function accountBalanceAfterPurchase (purchaseAmount: number): number {
    const mod = purchaseAmount % 10;
    if (mod > 4) {
        purchaseAmount += 10 - mod;
    } else {
        purchaseAmount -= mod;
    }
    return 100 - purchaseAmount;
}

File: src/2806.account-balance-after-rounded-purchase.2955/solution.java
class Solution {
    public int accountBalanceAfterPurchase(int purchaseAmount) {
        int mod = purchaseAmount%10;
        if(mod<5){
            purchaseAmount -= mod;
        }else{
            purchaseAmount += 10-mod;
        }
        return 100-purchaseAmount;
    }
}